java/arcs/core/data/proto/recipe.proto

```diff
@@ -25,11 +31,12 @@ message RecipeProto {
 message HandleProto {
   // Next Id: 5
   enum Fate {
-    CREATE = 0;
-    USE = 1;
-    MAP = 2;
-    COPY = 3;
-    JOIN = 4;
+    NOT_SET = 0;
```

Comment:
Do we need this? Is it the same as `UNRECOGNISED` in the generated enum? See https://developers.google.com/protocol-buffers/docs/reference/java-generated#enum-fields

Also, if we want to use 0, the convention is to use `UNSPECIFIED`: https://developers.google.com/protocol-buffers/docs/style#enums

Reply:
without it you don't have a distinction between unset and the value 0 (there is no hasX() method on new protos) which I find uncomfortable. Ale the explicitly set default value (e.g. 'create' fate) does not show up when printing the proto, which is also uncomfortable.

Updating to UNSPECIFIED.

Reply:
actually, if I leave the value = 0 entirely then UNSPECIFIED becomes UNRECOGNISED I think, moving to that.

Reply:
ah, no...
```
recipe.proto:34:14: The first enum value must be zero in proto3.
```